recipe/forge_test.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```

Comment:
Why you "hide" testing in this file?

Reply:
The script contains environment variables for `SSL_CERT_FILE` and `REQUESTS_CA_BUNDLE`, without unsetting them beforehand, the downloader tests fail. So I used the solution used by people [here](https://github.com/conda-forge/mosfit-feedstock/issues/23#issuecomment-349016207) for running the test suite for Linux.